### ISSUE TITLE
In VPL, misleading warning when switching to simple mode

### ISSUE THREAD

**Issue by ypiguet-epfl** [aseba-community/aseba#832]
In Aseba Studio, VPL, when switching from advanced mode to simple mode, a warning message is displayed, which reads "You are currently using states. Returning to simple mode will discard any state filter or state setting card. Are you sure you want to continue?" This is misleading, because states are not the only elements which are discarded. So are blocks or block settings specific to the advanced mode, such as the timer or the accelerometer.

**Comment by stephanemagnenat** [comment]
I agree, this message was created before these blocks were added. It should be updated.
